x-pack/plugins/actions/server/authorization/actions_authorization.ts

```diff
@@ -73,10 +80,21 @@ export class ActionsAuthorization {
           ? operationAlias[operation](authorization)
           : [authorization.actions.savedObject.get(ACTION_SAVED_OBJECT_TYPE, operation)];
 
-        const { hasAllRequested } = await checkPrivileges({
-          kibana: [...privileges, ...additionalPrivileges],
+        privileges.push(...additionalPrivileges);
+
+        const privilegesCheckResponse = await checkPrivileges({
+          kibana: privileges,
         });
-        if (!hasAllRequested) {
+
+        if (!privilegesCheckResponse.hasAllRequested) {
+          logger.info(
```

Comment:
Since we are now introducing the check of privileges that are outside of the Stack Management > Actions and Connectors, I felt it was best to log authz failures and dump the check on privileges - which will point to the missing privileges a user needs. I added it as `info()`, but let me know if you feel it should be something else (`.warn()`? or `.debug()`?)